roles/get_image_hash/defaults/main.yml

```diff
@@ -3,20 +3,32 @@ controller_tag: "{{ ai_version }}"
 installer_agent_tag: "{{ ai_version }}"
 installer_tag: "{{ ai_version }}"
 
-images_to_get_hash_for:
-  release:
-    image: quay.io/openshift-release-dev/ocp-release
-    tag: "{{ release_tag | default(openshift_full_version + '-x86_64') }}"
+ocp_release_versions_map_all:
+  '4.6': 4.6.16
+  '4.7': 4.7.33
+  '4.8': 4.8.14
+  '4.9': 4.9.0
+
+get_release_images: true
+get_all_release_version: false
+
+ocp_release_versions_map: "{{ get_all_release_version | ternary(ocp_release_versions_map_all, {openshift_version: ocp_release_versions_map_all[openshift_version]}) }}"
+
+release_image_url: "quay.io/openshift-release-dev/ocp-release"
+
+non_release_images:
```

Comment:
What's the distinction between non release and release images?

Reply:
It was just to distinguish that aren't the release image. perhaps `assisted_installer_images` would be better as that explains what they are used for more accurately.

Reply:
So `release_image` is referring to an OCP release, while non_release_image is for any other image?

Reply:
Yes so we could have multiple release images the other will be singular. I think the `assisted_installer_images` is a good name for the other subset and it people want to add other subsets... well is ansible...